vtm/src/org/oscim/map/Animator.java

```diff
@@ -224,6 +226,15 @@ public void animateZoom(long duration, double scaleBy,
 
     public void animateFling(float velocityX, float velocityY,
                              int xmin, int xmax, int ymin, int ymax) {
+        // Smooth fling: Uses velocity to calculate duration.
+        // Increase velocity with 500 as reference, cause duration decelerates animation.
+        float duration = (Math.abs(velocityX) + Math.abs(velocityY)) / 2;
```

Comment:
Don't understand the meaning here.
How velocity is named as duration?

Reply:
An example: A car brakes. The higher the velocity the longer it needs to stay steady. So the duration increases if velocity increases.
You can change comments of course if you have a better description :)

Reply:
Probably we can rename just the variable.

Reply:
What would you propose as variable name? Would this replace the calculations? 
Here the coefficient of friction is not given, so the calculations simulate it.

Reply:
Calculations could remain, can change name to "mean" during merge?

Reply:
Yes ˋmeanˋ would describe it better. ;D